src/components/HeroHome/_HeroHome.scss

```diff
@@ -1,3 +1,19 @@
+.coa-HeroHome-container {
+  img {
+    position: absolute;
+    z-index: -1;
+    height: 26rem;
+    width: 100%;
+    object-fit: cover;
+    @include media($coa-medium-screen) {
+      height: 40rem;
+    }
+    @include media($coa-large-screen) {
+      height: 44rem;
```

Comment:
seeing magic numbers on lines 5, 9, and 12 here. We've been managing those by assigning them as variables in this Sass file. https://github.com/cityofaustin/janis/blob/d4189a039396cc84a07c6fdc8281206c1fc7c290/src/css/base/_variables.scss#L86

maybe @ifsimicoded has more thoughts on that.